src/scripts.js

```diff
@@ -70,7 +70,13 @@ function isScriptHashInput(script, allowIncomplete) {
   if (!Buffer.isBuffer(lastChunk)) return false
 
   var scriptSig = Script.fromChunks(script.chunks.slice(0, -1))
-  var scriptPubKey = Script.fromBuffer(lastChunk)
+  var scriptPubKey
+
+  try {
+    scriptPubKey = Script.fromBuffer(lastChunk)
+  } catch (e) {
```

Comment:
This is an interesting concept, how would this be any different than returning a `console.debug` for every other instance where it returned false?